ts/Extensions/DataGrouping.ts

```diff
@@ -1012,6 +1022,26 @@ seriesProto.generatePoints = function (): void {
     this.groupedData = this.hasGroupedData ? this.points : null;
 };
 
+// When all series are processed, calculate the group pixel width and then
+// if this value is different than zero apply groupings.
+addEvent(Axis, 'afterProcessData', function (): void {
+    const axis = this,
+        series = axis.series;
+
+    series.forEach(function (series): void {
+        // Reset the groupPixelWidth, then calculate if needed.
+        series.groupPixelWidth = null as any; // #2110
+
+        series.groupPixelWidth = axis.getGroupPixelWidth && axis.getGroupPixelWidth();
```

Comment:
Simplify the assignment and get rid of the any cast (if we're not using `null` as a special meaning, in which case we need to add typing for it).

`series.groupPixelWidth = (axis.getGroupPixelWidth && axis.getGroupPixelWidth()) || void 0;`

Reply:
I agree that the `null as any` doesn't look nice.. 
I will change it to `void 0` however I can't simplify that assignment because before using the `getGroupPixelWidth` method the `groupPixelWidth` should be reset.